_conferences/droidcon-cluj-2019.md

```diff
@@ -0,0 +1,11 @@
+---
+name: "Droidcon Transylvania"
+website: http://www.droidcon.ro/
+location: Cluj-Napoca, Romania
+
+date_start: 2019-10-04
+date_end:   2019-10-05
+
+cfp_end:   2019-09-01  # Optional
```

Comment:
Both a start and end date needed for this to display, but since it's in the past you can just delete these cfp keys

Reply:
Yup, you're right. I'll remove the line.